packages/login/src/vaadin-login-form.js

```diff
@@ -60,69 +57,36 @@ class LoginForm extends LoginFormMixin(ElementMixin(ThemableMixin(PolylitMixin(L
     return 'vaadin-login-form';
   }
 
-  /**
-   * @protected
-   * @override
-   */
-  createRenderRoot() {
-    return this;
+  static get styles() {
+    return css`
+      :host {
+        display: block;
+        max-width: max-content;
```

Comment:
Added this to make centering easier by setting `margin: 0 auto` on the form element.
This seems like a better option than using `display: inline-block` to me.